### DISCUSSION TITLE
Quote reply doesn't work as in issues

### DISCUSSION THREAD

**Discussion by markcmiller86** [community/community#4150] [Discussions]
In issues, I can select a *portion* of text in a comment, go to the three dots in upper right (`...`) and get an option to `Quote reply` and then what I get is a new comment primed with *just* the text I selected quoted.

That doesn't appear to work in replies in discussions.

**Comment by Levi-Lesches** [comment]
> ment, go to the three dots in upper right (`...`) and get an option to `Quote reply` and then what I get is a new comment primed with _just_ the text I selecte

I think it does!

**Comment by markcmiller86** [comment]
Not in my experience. Sometimes it does. Most times it doesn’t

Try this...go here... https://github.com/visit-dav/visit/discussions/5733#discussioncomment-787717

Triple click on the line that has “I don’t know if there’s better workarounds...” to select that line (triple click winds up selecting a tad more like to emoji button). Then try to ‘quote reply’ and see if it works for you. It doesn’t for me in Safari...wait...even that does appear to work for me...now.

Hmmm...there is some sensitivity here because it often does not work for me but now as I play with it, it appears to. I wonder if it has something to do with whether its been primed in the past. Even in issues, sometimes the behavior I get is to quote the entire text, not just the selected text.

As I say above, this hasn’t been my experience and I am on GitHub.com many times throughout each day.

Mark



"Levi Lesches" wrote:


ment, go to the three dots in upper right (...) and get an option to Quote reply and then what I get is a new comment primed with just the text I selecte

I think it does!

**Reply by Levi-Lesches** [reply]
Could it be a sort of misclick where the selected text is de-selected right before or as you click "Quote reply"?

**Comment by markcmiller86** [comment]
Yes, surely it could. But, if it is, it seems a tad too likely of an occurrence. Either that or I seem to have an uncharacteristically common way of running into it.

Mark


"Levi Lesches" wrote:


Could it be a sort of misclick where the selected text is de-selected right before or as you click "Quote reply"?
